src/games/gamesContainer.py

```diff
@@ -88,7 +88,7 @@ def findGameByUuid(self, uuid):
 
     def addGame(self, game):
         """Add a game to the list"""
-        if not game in self.games :
+        if game not in self.games :
```

Comment:
Just fixing them as we see them will reduce them slowly but surely, if you want to do a full scale commit for it that is fine too.